app/controllers/liabilities_controller.rb

```diff
@@ -0,0 +1,27 @@
+class LiabilitiesController < ApplicationController
+  def pay
+    operation = CreatePayment.new(params[:id])
+
+    target = unless operation.error?
```

Comment:
Already done, just haven't pushed it yet. Rubocop picked me up on that one when the code coverage passed.